metrics/healtstatus.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// RecordHealthStatusEntry
+// Records the health status of a client.
+// uses the same senderLabel as signal_strength.go
+func RecordHealthStatusEntry(senderClientName string, healthStatus float64) {
```

Comment:
I would use name the param like this:
```suggestion
func RecordHealthStatusEntry(clientName string, healthStatus float64) {
```